validation.go

```diff
@@ -23,12 +23,14 @@ type objectValidation struct {
 	Violations *validationViolationSet
 }
 
-func validatePodSpec(validation *objectValidation, podSpec *corev1.PodSpec, config *config) {
+func validatePodSpec(validation *objectValidation, podMetadata *metav1.ObjectMeta, podSpec *corev1.PodSpec, config *config) {
 	for _, container := range podSpec.Containers {
 		validateContainerResources(validation, fmt.Sprintf("Container %s", container.Name), &container, config)
+		validateContainerSecurityContext(validation, podMetadata, fmt.Sprintf("Container %s", container.Name), &container, config)
 	}
 	for _, container := range podSpec.InitContainers {
 		validateContainerResources(validation, fmt.Sprintf("Init container %s", container.Name), &container, config)
+		validateContainerSecurityContext(validation, podMetadata, fmt.Sprintf("Container %s", container.Name), &container, config)
```

Comment:
Looks like a copy-paste error: the target description should read `"Init container ..." here.
Also, I'd extract that string to a variable, since it's being used two times. (The same goes for the previous code block dealing with normal containers)

Reply:
Correct. Extracted to a variable.